src/main/java/uk/gov/beis/els/api/categories/rangehoods/RangeHoodsApiController.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.beis.els.api.categories.rangehoods;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.beis.els.categories.internetlabelling.service.InternetLabelService;
+import uk.gov.beis.els.categories.rangehoods.model.RangeHoodsForm;
+import uk.gov.beis.els.categories.rangehoods.service.RangeHoodsService;
+import uk.gov.beis.els.model.ProductMetadata;
+import uk.gov.beis.els.service.DocumentRendererService;
+
+@RestController
+@RequestMapping("${api.v1.base_path}/range-hoods")
```

Comment:
Nit: we can remove `/range-hoods` since its the only item in this category. So the path can just be `/api/v1/range-hoods/energy-label` rather than `/api/v1/range-hoods/range-hoods/energy-label`

Reply:
done ;)